lambdas/src/utils/TheGraphClient.ts

```diff
@@ -81,15 +143,26 @@ export class TheGraphClient {
     `
   }
 
-  public findWearablesByOwner(owner: EthAddress): Promise<WearableId[]> {
+  /**
+   * Given an ethereum address, this method returns all wearables from ethereum and matic that are asociated to it.
+   * @param owner
+   */
+  public async findWearablesByOwner(owner: EthAddress): Promise<WearableId[]> {
+    const ethereumWearables = await this.getWearablesByOwner('collectionsSubgraph', owner)
+    const maticWearables = await this.getWearablesByOwner('maticCollectionsSubgraph', owner)
```

Comment:
We could make these requests in parallel:

```suggestion
    const ethereumWearablesPromise = this.getWearablesByOwner('collectionsSubgraph', owner)
    const maticWearablesPromise = this.getWearablesByOwner('maticCollectionsSubgraph', owner)
    const [ethereumWearables, maticWearables] = await Promise.all([ethereumWearablesPromise, maticWearablesPromise])
```